### PR TITLE
Use 'en' country/city translations as final fallback - VPN-4201

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: src/apps/vpn/serveri18n.cpp [hunk 1]
```diff
@@ -154,6 +154,13 @@ QString translateItemWithLanguage(const QString& languageCode,
     }
   }
 
+  // Let's use 'en' translation, which is still better than what we obtain from
+  // Guardian.
+  result = s_items.value(itemKey("en", countryCode, cityName));
+  if (!result.isEmpty()) {
```
Comment (brizental) [inline src/apps/vpn/serveri18n.cpp:160]: If this is the final fallback how come it can still be empty?

Comment (bakulf) [reply src/apps/vpn/serveri18n.cpp:160]: because we don't have the 'en' translation for all the city/countries.

Comment (brizental) [reply src/apps/vpn/serveri18n.cpp:160]: What is wrong with what we obtain from Guardian?

Comment (bakulf) [reply src/apps/vpn/serveri18n.cpp:160]: A lot of things are wrong. For instance they say: Osaka instead of Ōsaka.

**Review by brizental — APPROVED** [review]

File: src/apps/vpn/serveri18n.cpp (hunk 1 above)
Comment (brizental) [reply src/apps/vpn/serveri18n.cpp:160]: Should we file a follow up to fix that then?

Comment (bakulf) [reply src/apps/vpn/serveri18n.cpp:160]: that is from Mullvad. There is nothing we can change...

**Pull request merged**